### DISCUSSION TITLE
End of support for Samsung DeX

### DISCUSSION THREAD

**Discussion by viking1304** [Homebrew/discussions#2457] [Casks] [answered]
I just got a notification that new version of Samsung DeX is available, and this is the alert that I got after upgrade:

We're ending support for Samsung DeX for Mac OS in January 2022. You can still use Samsung DeX if it's already installed, but there will be no customer support or updates, and you won't he able to download DeX for Mac OS.

Since it wouldn't be possible to download any version of DeX officially, this formula should be deprecated in January.

I am just not sure should this be bumped at this moment or not. Looks pointless to me. Maybe I am just too disappointed in Samsung at this moment.

**Comment by neutric** [comment, marked as answer]
Thanks for reporting! I bumped it for `homebrew-cask` and will also discontinue/delete the cask come January.

**Reply by neutric** [reply]
=> https://github.com/Homebrew/homebrew-cask/pull/116823

Please mark this as answered.
